src/components/calcite-loader/calcite-loader.scss

```diff
@@ -1,7 +1,7 @@
 :host {
-  --calcite-loader-spot: #{$ui-blue};
-  --calcite-loader-spot-light: #{$v-bb-160};
-  --calcite-loader-spot-dark: #{$h-bb-070};
+  --calcite-loader-spot: var(--calcite-ui-blue);
```

Comment:
I think these need to pass the previous values - if you look at the demo page this seems to break the animation

Reply:
fixed, take another look.